app/migrations/0003_tweet.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2016-10-30 23:14
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('app', '0002_auto_20161030_1512'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Tweet',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('text', models.CharField(max_length=256)),
```

Comment:
From the sample of 100 tweets I had on my computer I got this distribution for string length:
   Min.  1st Qu.  Median    Mean  3rd Qu.    Max. 
   19.0    87.5    108.0        105.6   131.5      156.0

Looks like we could get more than 140.
ex: "Ridley Scott will receive the American Cinematheque Award. Matt Damon, Geena Davis &amp; more told me stories about him: https://t.co/1uyVXMqIyz" - 144 chars